tests/unit/test_computations.py

```diff
@@ -0,0 +1,112 @@
+"""Test the function of computations module."""
+
+import numpy as np
+import pytest
+
+from wristpy.core import computations, models
+from wristpy.io.readers import readers
+
+SIGNAL_LENGTH = 20
+EPOCH_LENGTH = 5
+
+
+def test_moving_mean_epoch_length_is_negative() -> None:
+    """Test error if the epoch length is negative."""
+    tmp_data = np.array([1, 2, 3])
+    time = readers.unix_epoch_time_to_polars_datetime(np.array([1, 2, 3]), "s")
+    tmp_Measurement = models.Measurement(measurements=tmp_data, time=time)
+
+    with pytest.raises(ValueError):
+        computations.moving_mean(tmp_Measurement, epoch_length=-EPOCH_LENGTH)
```

Comment:
If all that's relevant here is that EPOCH_LENGTH is a negative number, just write -1 or some such; in this set-up I'd have to scroll up to figure out what this is.